src/primitives/types.ts

```diff
@@ -42,7 +44,10 @@ export interface NavigableProps extends NodeProps {
   wrap?: boolean;
 }
 
-// @ts-expect-error animationSettings is not identical - weird
+declare module '@lightningtv/core' {
+  interface NodeProps extends NavigableProps {}
+}
+
```

Comment:
We don't want these on every node - only Row & Column currently have onSelectedChanged

Reply:
ok so it's just a weird trick to pass the callback to handleNavigation?
I don't like that, but I guess there is no point to extend the global interface